Source/Timeline/TimelineContainer.swift

```diff
@@ -15,6 +15,7 @@ public class TimelineContainer: UIScrollView, ReusableView {
   
   override public func layoutSubviews() {
     timeline.frame = CGRect(x: 0, y: 0, width: width, height: timeline.fullHeight)
+    timeline.updateAllDayView(by: contentOffset.y)
```

Comment:
I'd rename it to `setAllDayViewOffset(by:......` to give a little bit more insight into what the function does.

Reply:
I feel the same :)